File: articles/value-proposition-canvas-and-the-communication-of-relevance/fortran/value_relevance_score.f90
program value_relevance_score
  implicit none
  real :: score
  score = (0.91 + 0.84 + 0.86 + 0.76 + 0.90 + 0.88) / 6.0
  print '(A,F5.3)', 'Value relevance score: ', score
end program value_relevance_score

File: articles/value-proposition-canvas-and-the-communication-of-relevance/cpp/value_relevance_score.cpp
#include <iostream>
#include <iomanip>

int main() {
    double score = (0.91 + 0.84 + 0.86 + 0.76 + 0.90 + 0.88) / 6.0;
    std::cout << "Value relevance score: " << std::fixed << std::setprecision(3) << score << "\n";
    return 0;
}
